com.unity.ml-agents/Runtime/Communicator/GrpcExtensions.cs

```diff
@@ -96,14 +96,14 @@ public static BrainParametersProto ToProto(this BrainParameters bp, string name,
         {
             var brainParametersProto = new BrainParametersProto
             {
-                VectorActionSize = { bp.VectorActionSize },
-                VectorActionSpaceType = (SpaceTypeProto)bp.VectorActionSpaceType,
+                VectorActionSizeDeprecated = { bp.VectorActionSize },
+                VectorActionSpaceTypeDeprecated = (SpaceTypeProto)bp.VectorActionSpaceType,
                 BrainName = name,
                 IsTraining = isTraining
             };
             if (bp.VectorActionDescriptions != null)
             {
-                brainParametersProto.VectorActionDescriptions.AddRange(bp.VectorActionDescriptions);
+                brainParametersProto.VectorActionDescriptionsDeprecated.AddRange(bp.VectorActionDescriptions);
```

Comment:
nit: I don't think we should deprecate ActionDescriptions (unless we're replacing them with more specific ContinuousActionDescriptions , etc). We don't currently use them, but I think they will be helpful for debugging in the future, especially now that we can have multiple actuators and action types.

Reply:
I changed this because we won't use it anywhere other than deprecated cases since it's for VectorAction, and if we ever need a description we should add Continuous(/Discrete)ActionDescriptions